src/storage/index/b_plus_tree.cpp

```diff
@@ -31,7 +32,7 @@ auto BPLUSTREE_TYPE::IsEmpty() const -> bool { return true; }
  * @return : true means key exists
  */
 INDEX_TEMPLATE_ARGUMENTS
-auto BPLUSTREE_TYPE::GetValue(const KeyType &key, std::vector<ValueType> *result, Transaction *transaction) -> bool {
+auto BPLUSTREE_TYPE::GetValue(const KeyType &key, std::vector<ValueType> *result, Transaction *txn) -> bool {
```

Comment:
I think we should provide more instructions here by defining the prototype of GetHelper / PutHelper in refsol. We should at least show how to use `Context`.

Reply:
see slack thread, my comment to this contains spoilers about refsol lmao

Reply:
Yeah we should put something in the writeup. The refsol didn't implement it in a recursive way 😂. Mark as resolved.